src/bootstrap/configure.py

```diff
@@ -467,5 +467,5 @@ def configure_section(lines, config):
     f.write(contents)
 
 p("")
-p("run `python {}/x.py --help`".format(rust_dir))
+p("run `python3 {}/x.py --help`".format(rust_dir))
```

Comment:
This'll be my last comment for this review round on this -- but please don't replace `python` with `python3`.